### PR TITLE
fixed visited link colour

### PR DESCRIPTION
**Pull request by rohitmore07**
Closes #.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/assets/stylesheets/src/shared/_scaffolds.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $footer-min-height: 117px;
 $navbar-min-height: 56px;
+$visited-link-color: #044e92;
```
Comment (ksnitsky) [inline app/assets/stylesheets/src/shared/_scaffolds.scss:3]: Move that color to `_colors.scss`, please

**Review by ksnitsky — CHANGES_REQUESTED** [review]
Update the branch to the latest version of `migration-to-tabler`.
We removed these styles in one of the first commits of this branch, so you may have to add them again in your PR.

File: app/assets/stylesheets/src/shared/_scaffolds.scss (hunk 1 above)
Comment (rohitmore07) [reply app/assets/stylesheets/src/shared/_scaffolds.scss:3]: Sure!

**Comment by rohitmore07** [comment]
Done @ksnitsky

**Review by ksnitsky — COMMENTED** [review]
You need to make your branch from our `migration-to-tabler` branch https://github.com/freeats/freeats/pull/1 and make changes on top of it, not on top of `main`. 

In this branch, some of the styles you made changes to have been deleted, so you have 2 merge conflict notices in the merge block.

**Pull request closed without merging**